documentation/how-to-use-appyx/quick-start.md

```diff
@@ -1,5 +1,14 @@
 # Quick start guide
 
+!!! info
```

Comment:
Do these render special characters when using the doc tool?

Reply:
Yep, https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
sounds good!